SpringProject/src/main/java/com/gradproject2019/conferences/controller/ConferenceController.java

```diff
@@ -14,8 +15,15 @@
     private ConferenceService conferenceService;
 
     @GetMapping(path = "/conferences")
-    public @ResponseBody
-    Iterable<Conference> getAllUsers() {
+    @ResponseBody
+    public Iterable<Conference> getAllUsers() {
         return conferenceService.listConferences();
     }
+
+    @GetMapping(path = "/conferences/{id}")
+    @ResponseBody
```

Comment:
No need for ResponseBody annotation. If your jackson mapping is properly configured in your project it will do conversion by default.